pkg/virt-controller/services/template_test.go

```diff
@@ -173,6 +173,37 @@ var _ = Describe("Template", func() {
 				Expect(pod.Spec.Subdomain).To(BeEmpty())
 			})
 		})
+		Context("with SELinux types", func() {
+			It("should run under the SELinux type container_t if none specified", func() {
+				vmi := v1.VirtualMachineInstance{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "testvmi", Namespace: "default", UID: "1234",
+					},
+					Spec: v1.VirtualMachineInstanceSpec{Volumes: []v1.Volume{}, Domain: v1.DomainSpec{}},
+				}
+				pod, err := svc.RenderLaunchManifest(&vmi)
+				Expect(err).ToNot(HaveOccurred())
+				if pod.Spec.SecurityContext != nil && pod.Spec.SecurityContext.SELinuxOptions != nil {
+					Expect(pod.Spec.SecurityContext.SELinuxOptions.Type).To(Equal(""))
```

Comment:
maybe, `.To(BeEmpty())` instead?

Reply:
I wasn't sure `.To(BeEmpty())` would work on a string. The rest of the file seems to use that only on arrays, and `""` on strings.